pom.xml

```diff
@@ -94,6 +94,10 @@
 
     <!-- Release: enable publication to Bintray -->
     <artifactsToPublish>${project.groupId}:sonar-java-plugin:jar</artifactsToPublish>
+
+    <!-- currently fails with "javadoc: error - The code being documented uses modules but the packages defined in https://docs.oracle.com/javase/8/docs/api/ are in the unnamed module." -->
+    <!-- FIXME this should be dropped when jdk bug has been fixed, as with this parameter we STOP PRODUCING Javadoc with releases-->
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Take the required action to fix the issue indicated by this "FIXME" comment. ([xml:S1134](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=xml%3AS1134))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=2392&issues=AWhchtH2Uv3CKBQtewc_&open=AWhchtH2Uv3CKBQtewc_)

[ISSUE_KEY]: AWhchtH2Uv3CKBQtewc_